tools/bundle-doc/index.js

```diff
@@ -55,21 +55,18 @@ processDir(docImagesRoot);
  * much easier than checking ~1400 lines.
  */
 const autogeneratedContent = `// THIS FILE IS AUTOGENERATED. DO NOT EDIT IT MANUALLY.
-${docFiles.map((f, i) => `import file${i} from "./doc/${f}?raw";`).join("\n")}
-${nsDocFiles.map((f, i) => `import nsDoc_${f.replaceAll(".", "_")} from "../../markdown/${f}?raw";`).join("\n")}
-${docImagesFiles.map((f) => `import docImages_${f.replaceAll(/\.|-/g, "_")} from "./images/${f}";`).join("\n")}
-
+${docFiles.map((f, i) => `import file${i} from "./doc/${f}?raw";\n`).join("")}
+${nsDocFiles.map((f) => `import nsDoc_${f.replaceAll(".", "_")} from "../../markdown/${f}?raw";\n`).join("")}
+${docImagesFiles.map((f) => `import docImages_${f.replaceAll(/\.|-/g, "_")} from "./images/${f}";\n`).join("")}
```

Comment:
This is an aesthetic choice/trade-off. As it stands now, there is effectively an extra newline for each of these from the "natural" end-of-line after the statement. This is creating extra spacing between the groups, which can be a good thing, but it also means there would be extra spaces if any group were empty.

The alternative would be to have all of these on one line. That would fix the generated output, but the code would be harder to read. The generated file is not meant to be looked at, so I don't really care either way - I bring this up as a trade-off between the two styles, for the future.